scripts/perfmon

```diff
@@ -1066,13 +1064,13 @@ def main():
             vm_uuid_list = rrd_updates.get_uuid_list_by_objtype('vm')
 
             # Remove any monitors for VMs no longer listed in rrd_updates page
-            for uuid in vm_mon_lookup.keys():
+            for uuid in list(vm_mon_lookup.keys()):
```

Comment:
```suggestion
            for uuid in vm_mon_lookup:
```
ditto, AFIACS

Reply:
Re-checked the code.
The `list` is necessary for the code, for the code in the loop would modify the dictionary and would cause ` RuntimeError: dictionary changed size during iteration` in python3.
Will raise another PR to fix this.